src/libs/Navigation/AppNavigator/AuthScreens.js

```diff
@@ -113,12 +113,27 @@ class AuthScreens extends React.Component {
         // Load policies, maybe creating a new policy first.
         Linking.getInitialURL()
             .then((url) => {
-                if (this.shouldCreateFreePolicy(url)) {
-                    Policy.createAndGetPolicyList();
+                if (!url) {
```

Comment:
**question:** When would there be no `url` ? 

If there isn't one do we still want to call `Policy.getPolicyList()`?

Reply:
According to [the documentation](https://reactnative.dev/docs/linking#getinitialurl) "If the app launch was triggered by an app link, it will give the link url, otherwise it will give `null`." Thanks for pointing out that I'm not calling `Policy.getPolicyList()` in this case. I'm sure we still want to get the policy list if the app is opened regularly, so I will fix that.